website/docs/install/partials/_quickstart-intro.md

```diff
@@ -0,0 +1,37 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info Already running a node?
+
+This guidance is targeted at new users. If you're already running a node, see [Prepare for The Merge](../prepare-for-merge.md).
+
+:::
+
+<br />
+
+First, <strong>select a quickstart configuration</strong>. If you're looking for the simplest configuration, select `Geth` and `IPC`:
```

Comment:
cc @nisdas (I'll @ you for the IPC bits to make your review a little easier)

Reply:
I think nethermind supports IPC too ? Did you ever manage to get IPC working on nethermind

Reply:
I'll unlock this path in a separate PR :)